.github/workflows/publish.yaml

```diff
@@ -0,0 +1,79 @@
+name: Docker Image Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  EDGE: v1.14/alpine
+
+jobs:
+  publish:
+    name: Publish Docker Image
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [v1.14]
+        os: [alpine]
+    runs-on: ubuntu-latest
+    env:
+      EDGE_TAGS: ${{ fromJSON('["","type=raw,value=edge"]')[format('{0}/{1}', matrix.version, matrix.os) == env.EDGE] }}
```

Comment:
Can you document the expected behaviour? It took me a while to figure out what is going on with the various conditions so rather have it clear - plus I can then review your design vs the implementation.